common/src/constants/grade-boundaries.ts

```diff
@@ -129,12 +129,8 @@ export const MUSECA_GBOUNDARIES = MakeGradeBoundaries<GetEnumValue<"museca:Singl
 	傑G: 1_000_000,
 });
 
-export const MAIMAIDX_GBOUNDARIES = MakeGradeBoundaries<GetEnumValue<"maimaidx:Single", "grade">>({
-	D: 0,
-	C: 10,
-	B: 20,
-	BB: 40,
-	BBB: 60,
+const MAIMAI_COMMON_BOUNDARIES = {
```

Comment:
don't put this into a common thing, it adds indirection in the codebase for no reason. Just write it out twice; they're separate games.